exercises/10_get_file_size.py

```diff
@@ -42,33 +42,36 @@ class NumpySeqHandler:
     def __init__(self, filename, root=''):
         self._name = os.path.join(root, filename)
 
+
     def __call__(self, index):
         return np.load('{}_{}.npy'.format(self._name, index))
 
+
     def get_file_list(self, datum_kwarg_gen):
-        "This method is optional. It is not needed for access, but for export."
+        #This method is optional. It is not needed for access, but for export.
         return ['{name}_{index}.npy'.format(name=self._name, **kwargs)
                 for kwargs in datum_kwarg_gen]
 
+
+    def get_file_size(self, datum_kwarg_gen):
+        resource = db.reg.resource_given_datum_id(datum_kwarg_gen['datum_id'])
```

Comment:
since we already have the filename in the class, we should just ignore everything and return from that, something like:
```
for datum in datum_gen:
    index = datum['datum_kwargs']['index']
    filename = f'{self._name}_{index}.npy'
    size = os.path.getsize(filename)
```
(so we don't need to retrieve the resource again from db)
try to re-use info from the class first when possible.

Also, we don't want any calls to `db` in the file handler. We want to make sure file handlers are not database specific.